substrate/primitives/state-machine/src/ext.rs

```diff
@@ -713,6 +713,7 @@ where
 }
 
 /// Implement `Encode` by forwarding the stored raw vec.
+#[allow(dead_code)]
```

Comment:
```suggestion
#[cfg(any(test, feature = "fuzzing"))]
```

Reply:
It is still used in two log statements, so guarding it with test and fuzzing wont work.